presto-tests/src/test/java/com/facebook/presto/tests/TestQueryPlanDeterminism.java

```diff
@@ -39,7 +40,7 @@
 
     protected TestQueryPlanDeterminism()
     {
-        super(TestQueryPlanDeterminism::createLocalQueryRunner);
+        super(TestQueryPlanDeterminism::createLocalQueryRunner, FeatureSelection.all().without(ENGINE_ONLY).build());
```

Comment:
really? not the other way around (e.g: include engine/planner only) features?

Reply:
removed that change